fullnumlistdivisors/fullnumlistdivisors.py

```diff
@@ -0,0 +1,42 @@
+#!python3
+# script that prints first N numbers divisible by --divisors=3,5,77,...
+import argparse
+from typing import List, Dict, Set
+#from sortedcontainers import SortedSet
+
+parser = argparse.ArgumentParser(description="The script prints first N numbers divisible by 3 or 5")
+parser.add_argument("-n", required=True, type=int, help="integer to show the list of numbers divisible by 3 or 5 ")
+parser.add_argument("-d", "--divisors", required=True, help="")
+args = parser.parse_args()
+
+def unique_divisors_set(unique_set: Set[int], dividor: int) -> Set[int]:
+    unique_set.add(dividor)
+    return unique_set
+
+
+def main():
+    error_check_pass = False
+    unique_set: Set[int] = set()
+    divisors = args.divisors.split(",")
+    for divisor in divisors:
+        try:
+            unique_set = unique_divisors_set(unique_set, int(divisor))
+            error_check_pass = True
```

Comment:
No need for this line, you can initialize `error_check_pass` with `True` and then you only need line `27`, e.g. `error_check_pass` is `True` **unless** at least one of divisors is not integer.

Reply:
Got it!

error_check_pass = True

...

if not (error_check_pass and unique_set):
        return